src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -68,7 +68,7 @@ public static void WriteNumberTooLargeScientific(bool indented)
             // confirm the printing precision of double, like
             //
             //double oneQuarticGoogol = double.Parse(OneQuarticGoogol);
-            //Assert.NotEqual(OneQuarticGoogol, oneQuarticGoogol.ToString("R"));
+            //Assert.NotEqual(OneQuarticGoogol, oneQuarticGoogol.ToString(JsonTestHelper.DoubleFormatString));
```

Comment:
Is there a reason why this is commented out?

Reply:
Because it was originally commented out and I was only fixing up the existing state.

Reply:
@bartonjs, do you know why?

Reply:
The comments say "one could confirm the printing precision...".  1e400 (a quartic-googol) is beyond the range of double.  It's a legal JSON Number value, but in double it's Infinity.

This test shows that JsonElement.WriteValue is not using double as a transport, or we'd lose data, even with R (or R17, or whatever).  The Assert.NotEqual sample is just "this is why this test exists", not "this is what the test tests"

Reply:
We should either uncomment it or delete it.

Reply:
It's not commented out code, it's a sample-code-comment explaining a text-comment.

But deleting it is fine.

Reply:
I'd read the comment. It says you can do the following for further validation. Either we need the validation or we don't.

Reply:
We already have tests that validate the behavior of too large/small values

Reply:
> We already have tests that validate the behavior of too large/small values

Getting them as a `double` from the payload, yes.  The unique value of this test is that it shows that JsonDocument.Parse, then JsonElement.WriteValue preserves the value even when it's out of range of double (or, in the .NET Core 3.0 case, is equivalent to Infinity).

The extra validation would assert that this test is actually testing that scenario, vs that the scenario works.

So, don't delete the test.  The comment block, including the commented out example, can be uncommented... except that it would then have to tolerate throwing on .NET Framework.

Since the purpose of the comment was to explain and save time in maintenance, but has cost significant time in maintenance, just delete it.

Reply:
I changed the comment to be:
```cs
            // This just validates we write the literal number 1e400 even though it is too
            // large to be represented by System.Double and would be converted to
            // PositiveInfinity instead (or throw if using double.Parse on frameworks
            // older than .NET Core 3.0).
```